handler.go

```diff
@@ -57,6 +56,8 @@ type Handler struct {
 	BroadcastHandler BroadcastHandler
 	StatusHandler    StatusHandler
 
+	StaticFileSystem StaticFileSystem
```

Comment:
I think we should just call this "FileSystem" - staticness is an implementation detail.